app/views/planning_applications/_assess_proposal.html.erb

```diff
@@ -2,34 +2,46 @@
   <% if (current_user.assessor? || current_user.admin?)%>
     <li>
       <h2 class="app-task-list__section">
-        <span class="app-task-list__section-number">Make recommendation</span>
+        <% if @planning_application.correction_requested? %>
+          <span class="app-task-list__section-number">Make corrections</span>
+        <% else %>
+          <span class="app-task-list__section-number">Make recommendation</span>
+        <% end %>
       </h2>
       <ul class="app-task-list__items">
         <li class="app-task-list__item">
-          <% step_name = "Assess the proposal" %>
+          <% if @planning_application.correction?  %>
+            <% step_name = "Reassess the proposal" %>
+          <% else %>
+            <% step_name = "Assess the proposal" %>
+          <% end %>
```

Comment:
I think it would be great if we could we extract this functionality into a service and here we just initialise the name `step_name`. What do you think?

Reply:
I've added a few helper methods https://github.com/unboxed/bops/pull/100/commits/9985df90cee6fb5cd4c41c21dc356d7427c2f08d